src/main/java/uk/gov/hmcts/reform/pip/model/publication/ListType.java

```diff
@@ -63,7 +63,19 @@ public enum ListType {
     PHT_WEEKLY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES),
     GRC_WEEKLY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES),
     WPAFCC_WEEKLY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES),
-    UT_IAC_JUDICIAL_REVIEW_DAILY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES),
+    UT_IAC_JR_LONDON_DAILY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES),
+    UT_IAC_JR_MANCHESTER_DAILY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES,
+        UT_IAC_JR_LONDON_DAILY_HEARING_LIST,
+        "Upper Tribunal (Immigration and Asylum) Chamber - Judicial Review: Manchester Daily hearing List",
+        false, true),
+    UT_IAC_JR_BIRMINGHAM_DAILY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES,
+        UT_IAC_JR_LONDON_DAILY_HEARING_LIST,
+        "Upper Tribunal (Immigration and Asylum) Chamber - Judicial Review: Birmingham Daily hearing List",
+        false, true),
+    UT_IAC_JR_CARDIFF_DAILY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES,
+        UT_IAC_JR_LONDON_DAILY_HEARING_LIST,
+        "Upper Tribunal (Immigration and Asylum) Chamber - Judicial Review: Cardiff Daily hearing List",
```

Comment:
hearing needs to have uppercase H like other list types

Reply:
fixed